package.json

```diff
@@ -23,7 +23,6 @@
     "lint:fix": "eslint --ext .js,.jsx --fix .",
     "lint:examples": "eslint -c ./src/data/markdown/.eslintrc.js src/data/markdown",
     "lint:examples:fix": "eslint -c ./src/data/markdown/.eslintrc.js --fix src/data/markdown",
-    "lint:prose": "vale --glob='**/{docs,translated-guides/en}/**/*.md' ./src/data/markdown",
```

Comment:
@SilencerWeb, this command is necessary to validate it locally.

Reply:
@ppcano returned it back